frontend/uddhar-frontend/src/shared/components/ProfileHeader.jsx

```
@@ -0,0 +1,48 @@
+    <div className="w-full bg-white border-b border-gray-200 px-6 py-4 sticky top-0 z-5 shadow-sm">
+      <div className="flex justify-between items-center">
+        <div>
+          <h1 className="text-xl font-semibold text-gray-800">
+            Profile Information
+          </h1>
+        </div>
+        {!isEditing ? (
+          <button
+            onClick={() => setIsEditing(true)}
+            className="px-4 py-2 border border-transparent rounded-md shadow-sm text-sm font-medium text-white bg-blue-600 hover:bg-blue-700"
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)